vendor/github.com/docker/go-connections/nat/sort.go

```diff
@@ -58,7 +58,7 @@ func (s portMapSorter) Less(i, j int) bool {
 func SortPortMap(ports []Port, bindings PortMap) {
 	s := portMapSorter{}
 	for _, p := range ports {
-		if binding, ok := bindings[p]; ok {
+		if binding, ok := bindings[p]; ok && len(binding) > 0 {
```

Comment:
I see this is vendored, just trying to understand why we'd check `len(binding)` if we already know `bindings[p]` exists?

Reply:
Yah, since it's only used in the loop this doesn't make any real sense

Reply:
Scratch that previous comment, if there's nothing in binding we don't want to set bindings[p] at all, that's why.

Reply:
Oh there's the else in there.

Reply:
Also a single line after the for loop. My eye skipped right over both of them too for some reason.